### PR TITLE
Updates to include FOXSI-3

### PR DESCRIPTION
**Pull request by somusset**
The option for FOXSI-3 was added to a few files, as well as a correction to calculate altitude in level0 to level1.

### CODE REVIEW

**Comment by jtvievering** [comment]
For line 60 in foxsi_level0_to_level1.pro, I wonder if the default filename should be for the FOXSI-2 level 0 data since the default year is 2014?

File: proc/wsmr_data_to_hskp_level0.pro [hunk 1]
```diff
@@ -9,10 +9,11 @@
 ; use wsmr_data_to_level0.pro
 ;
 ; Inputs:
-;		FILENAME	File to process. Must be a WSMR .log file.
-;			   		Default is the 2012 Nov 2 flight data file.
 ;
 ; Keywords:	
+;		FILENAME	File to process. Must be a WSMR .log file.
+;			   		Default is the 2012 Nov 2 flight data file.
```
Comment (jtvievering) [inline proc/wsmr_data_to_hskp_level0.pro]: Remove this comment since the 2012 flight data is currently not set as the default.

**Comment by jtvievering** [comment]
In wsmr_data_to_hskp_level0.pro, lines 244-258 use appear to use information specific to the 2012 flight; add the capability to process data from 2014 and 2018.

File: proc/wsmr_data_to_level0.pro [hunk 2]
```diff
@@ -28,6 +27,9 @@
 ;		file = 'data_2012/foxsi_level0_data.sav'
 ;
 ; History:	
+;		2019-Oct-15	Sophie	Delete the default selection of filename. If we want a selection of the filename, 
+;						we can change this input into a keyword and select the filename using the year keyword
+;					Added file option for FOXSI-3
```
Comment (jtvievering) [inline proc/wsmr_data_to_level0.pro]: In description of keyword "year", include 2018 as an option.

File: proc/wsmr_data_to_level0.pro [hunk 3]
```diff
@@ -255,7 +262,7 @@ FUNCTION	WSMR_DATA_TO_LEVEL0, FILENAME, DETECTOR=DETECTOR, STOP=STOP, YEAR=YEAR
 	if year eq 2012 then begin
```
Comment (jtvievering) [inline proc/wsmr_data_to_level0.pro]: Do options for 2014 and 2018 need to be provided here as well?

File: proc/wsmr_data_to_hskp_level0.pro (hunk 1 above)
Comment (somusset) [reply proc/wsmr_data_to_hskp_level0.pro]: Done!

File: proc/wsmr_data_to_level0.pro (hunk 3 above)
Comment (jtvievering) [reply proc/wsmr_data_to_level0.pro]: ^ This is my only remaining question about this pull request. ^

**Comment by somusset** [comment]
The altitude is extracted in foxsi_level0_to_level1.pro
Therefore, this if statement could be removed here, but just in case it is used in someone's code, I left it. We could also decide that we want to have altitude data in the level 0 files, which is not the case here, except for FOXSI-1 because of these lines...
I added a comment in the code. We could merge this pull request and make an issue to discuss this further without delaying the merging of the pull request, what do you think?

**Comment by jtvievering** [comment]
That makes sense, and I don't feel strongly about having the altitude information in the level 0 data. The comments you added help to clarify the purpose of the code, and I think this pull request is ready to be merged!

**Pull request merged**